io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,411 @@
+app-id: io.github.pemsley.coot
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+command: coot
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
+  - --share=network
+
+modules:
+  - deps/pypi-dependencies.json
+
+  - name: fftw2
+    buildsystem: autotools
+    config-opts:
+      - --enable-shared
+      - --enable-float
+      - --disable-static
+    # build-commands:
+    #   - |
+    #     if [ "$(uname -m)" = "aarch64" ]; then
+    #       sed -i 's/-flat_namespace -undefined suppress/-undefined dynamic_lookup/g' configure
+    #     fi
+    #     ./configure
+    #     make
+    #     make install
+    sources:
+      - type: archive
+        url: https://fftw.org/fftw-2.1.5.tar.gz
+        sha256: f8057fae1c7df8b99116783ef3e94a6a44518d49c72e2e630c24b689c6022630
+
+  - name: mmdb2
+    buildsystem: autotools
+    config-opts:
+      - --enable-shared
+      - --disable-static
+    build-options:
+      cxxflags: "-g -O2"
+    sources:
+      - type: archive
+        url: https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/dependencies/mmdb2-2.0.22.tar.gz
+        sha256: a9646933ce9f34633e1ae4901f2383af0e5318d6490851328f5b6aa06195ab51
+
+  - name: libccp4
+    buildsystem: simple
```

Comment:
use autotools, `subdir: libccp4`, then put the config args as `config-opts` and drop build-commands.